web/src/app/modules/shared/components/smart/terminal/terminal.component.ts

```diff
@@ -13,18 +13,15 @@ import {
 } from '@angular/core';
 import { Terminal } from 'xterm';
 import { FitAddon } from 'xterm-addon-fit';
+import { v4 as uuidv4 } from 'uuid';
 import {
   TerminalOutputService,
   TerminalOutputStreamer,
 } from 'src/app/modules/shared/terminals/terminals.service';
 import trackByIdentity from 'src/app/util/trackBy/trackByIdentity';
-import {
-  TerminalDetail,
-  TerminalView,
-} from 'src/app/modules/shared/models/content';
+import { TerminalView } from 'src/app/modules/shared/models/content';
 import { WebsocketService } from '../../../services/websocket/websocket.service';
-import { SliderService } from 'src/app/modules/shared/slider/slider.service';
```

Comment:
There is some more cleanup for slider service. Mainly removing the remaining usage in `container.component.ts` and extra margin in the scss file then the service can be removed entirely.

Reply:
Great point, removed